author: Andres Salomon <dilinger@debian.org>
description: include debian node libs (needed for rollup)

This is strictly just needed for bullseye's rollup , but may be useful
later on when we drop more nodejs stuff.

--- a/third_party/node/node_modules.py
+++ b/third_party/node/node_modules.py
@@ -4,10 +4,13 @@
 # found in the LICENSE file.
 
 from os import path as os_path
+import os
 
 
 NODE_MODULES = os_path.join(os_path.dirname(__file__), 'node_modules')
 
+os.environ['NODE_PATH'] = '/usr/share/nodejs'
+
 
 def _path_in_node_modules(*args):
   return os_path.join(NODE_MODULES, *args)
@@ -22,7 +25,7 @@ def PathToHtmlMinifier():
 
 
 def PathToRollup():
-  return _path_in_node_modules('@rollup', 'wasm-node', 'dist', 'bin', 'rollup')
+  return _path_in_node_modules('rollup', 'dist', 'bin', 'rollup')
 
 
 def PathToSvgo():
